tensorflow/python/ops/ctc_ops.py

```diff
@@ -897,7 +897,7 @@ def ctc_loss_v3(labels,
     setting of preprocess_collapse_repeated=False, ctc_merge_repeated=True
   - Labels may be supplied as either a dense, zero-padded tensor with a
     vector of label sequence lengths OR as a SparseTensor.
-  - On TPU and GPU: Only dense padded labels are supported.
+  - On TPU: Only dense padded labels are supported.
   - On CPU: Caller may use SparseTensor or dense padded labels but calling with
```

Comment:
Change this to "On CPU and GPU:  Caller may..."

since (as noted by other commenters) using sparse labels will be significantly faster.